internal/client/agent/option.go

```diff
@@ -0,0 +1,41 @@
+package agent
+
+import "github.com/vdaas/vald/internal/config"
+
+type Option func(*agentClient)
+
+var (
+	defaultOptions = []Option{
+		WithAddr("0.0.0.0"),
```

Comment:
Client needs to connect server with port.

Reply:
oh:bow:
I will fix it
thank you for review.